src/redux/books/books.js

```diff
@@ -0,0 +1,35 @@
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+const idCounter = 0;
+const initialState = [];
+
+const remove = (state, bookID) => {
+  state.filter((book) => book.id !== bookID);
+};
+
+export default (state = initialState, action = {}) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [
+        ...state,
+        {
+          id: idCounter + 1,
```

Comment:
- [ ] Instead of creating id like this use `Math.floor(Math.random() * 100)`.

Reply:
Hi @Deepakdanger
I don't understand why you want me to implement these changes as you haven't attached any reason as to why the changes are necessary.